avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1311,6 +1352,13 @@ private Object toConnectData(Schema schema, Object value) {
   }
 
   public Schema toConnectSchema(org.apache.avro.Schema schema) {
+    return toConnectSchema(
+        schema, new ToConnectContext(new IdentityHashMap<>(), new HashSet<>()));
```

Comment:
we should probably just have the constructor handle instantiation of these, they should never be anything than new, empty maps.